### PR TITLE
Patch 3

### PR DESCRIPTION
**Pull request by keylimesoda**
Updating documentation to include duplicate subscription behavior.

### CODE REVIEW

File: concepts/change-notifications-delivery-event-hubs.md [hunk 1]
```diff
@@ -139,6 +139,8 @@ The main difference during subscription creation will be the **notificationUrl**
 - `secretname` - The name you gave to the secret when you created it. Can be found on the Azure Key Vault **Secrets** page.
 - `domainname` - The name of your tenant; for example, consto.onmicrosoft.com or contoso.com. Because this domain will be used to access the Azure Key Vault, it is important that it matches the domain used by the Azure subscription that holds the Azure Key Vault. To get this information, you can go to the overview page of the Azure Key Vault you created and click the subscription. The domain name is displayed under the **Directory** field.
 
+> [!NOTE]
+> Duplicate subscriptions are not allowed.  When a subscription request contains the same values for `changeType` and `resource` as an existing subscription, the request will fail with an HTTP error code `409 Conflict`, and a return message "Subscription Id <> already exists for the requested combination".
```
Comment (FaithOmbongi) [inline concepts/change-notifications-delivery-event-hubs.md:143]: - Apply the suggestion to the other file.
- Also, add this statement to the two subscription-post-subscriptions.md API ref files, in the "Response" section, after the "If successful, this method returns 201 Created response code and a subscription object in the response body." statement. Just a sentence, not the note alert type.

```suggestion
> Duplicate subscriptions are not allowed.  When a subscription request contains the same values for `changeType` and `resource` as an existing subscription, the request will fail with an HTTP error code `409 Conflict`, and the error message "Subscription Id <> already exists for the requested combination".
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @keylimesoda - I've left two suggestions. Please address those.

I've also pinged you internally and shared some recommendations.

**Review by keylimesoda — COMMENTED** [review]
Good changes.  Accepted.

**Comment by keylimesoda** [comment]
@FaithOmbongi with the acceptance of your latest changes, is this good to merge?

**Review by FaithOmbongi — APPROVED** [review]

**Pull request merged**